CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 1.34.0 - 2023-10-24
+- Feature: support `--device-locale` parameter for `maestro start-device`
+- Feature: add `centerElement` parameter for `scrollUntilVisible` to center on element after scrolling finished
```

Comment:
> to center on element after scrolling finished
I would word it like this: Center element will attempt to stop scrolling when the element is near the center of the screen